### DISCUSSION TITLE
How to build grasscutter game version 2.8

### DISCUSSION THREAD

**Discussion by IMrProYTI** [Melledy/Grasscutter#1492] [Support] [answered]
I want build project but i don't understand how i can change root of project for building.
I try to build development version but it's 2.7 game version (Also grasscutter-1.2.2-dev).
I will be grateful for any help :В

**Comment by 4Benj** [comment, marked as answer]
Even if you get it built, you need to do client patching and with the current PR, it requires UA patching which will have to be done manually since the Cultivation PR for client patching patches the metadata and not the UA.
The backrooms will be submitting a PR within the next 24 hours that is an alternative version of the current 2.8 which patches the metadata instead and implements other new features.

**Comment by IMrProYTI** [comment]
I have patched client, but I have the wrong build.
Guess I'll just wait for a new build of grass cutter with 2.8 support)
